### DISCUSSION TITLE
Override Filament-Support views

### DISCUSSION THREAD

**Discussion by tigusigalpa** [filamentphp/filament#3918] [Help] [answered]
Hi all! Filament has a **support package** and own views like _button, dropdowns_ etc. Is it possible to override views of this **support package**?
I putting files to (for example) `resources/views/vendor/support/components/button.blade.php` but not working.

**Comment by danharrin** [comment, marked as answer]
Try `/vendor/filament-support`

**Comment by ghost** [comment]
This does not work. It does not respond to overrides on 
/vendor/filament/support
/vendor/support
/vendor/filament-support
/filament/support
/filament-support
